[2022-11-04 11:52:28,237] {processor.py:153} INFO - Started process (PID=1702) to work on /opt/airflow/dags/groups/group_downloads.py
[2022-11-04 11:52:28,244] {processor.py:641} INFO - Processing file /opt/airflow/dags/groups/group_downloads.py for tasks to queue
[2022-11-04 11:52:28,256] {logging_mixin.py:115} INFO - [2022-11-04 11:52:28,256] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/groups/group_downloads.py
[2022-11-04 11:52:28,272] {logging_mixin.py:115} INFO - [2022-11-04 11:52:28,271] {dagbag.py:320} ERROR - Failed to import: /opt/airflow/dags/groups/group_downloads.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/groups/group_downloads.py", line 6
    with DAG(f"{parent_dag_id}.{child_dag_id")
                                             ^
SyntaxError: invalid syntax
[2022-11-04 11:52:28,275] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/groups/group_downloads.py
[2022-11-04 11:52:28,675] {processor.py:161} INFO - Processing /opt/airflow/dags/groups/group_downloads.py took 0.455 seconds
[2022-11-04 11:52:59,306] {processor.py:153} INFO - Started process (PID=1736) to work on /opt/airflow/dags/groups/group_downloads.py
[2022-11-04 11:52:59,308] {processor.py:641} INFO - Processing file /opt/airflow/dags/groups/group_downloads.py for tasks to queue
[2022-11-04 11:52:59,311] {logging_mixin.py:115} INFO - [2022-11-04 11:52:59,310] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/groups/group_downloads.py
[2022-11-04 11:52:59,335] {logging_mixin.py:115} INFO - [2022-11-04 11:52:59,328] {dagbag.py:320} ERROR - Failed to import: /opt/airflow/dags/groups/group_downloads.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/groups/group_downloads.py", line 6
    with DAG(f"{parent_dag_id}.{child_dag_id")
                                             ^
SyntaxError: invalid syntax
[2022-11-04 11:52:59,337] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/groups/group_downloads.py
[2022-11-04 11:52:59,672] {processor.py:161} INFO - Processing /opt/airflow/dags/groups/group_downloads.py took 0.373 seconds
[2022-11-04 11:57:07,119] {processor.py:153} INFO - Started process (PID=1747) to work on /opt/airflow/dags/groups/group_downloads.py
[2022-11-04 11:57:07,121] {processor.py:641} INFO - Processing file /opt/airflow/dags/groups/group_downloads.py for tasks to queue
[2022-11-04 11:57:07,130] {logging_mixin.py:115} INFO - [2022-11-04 11:57:07,130] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/groups/group_downloads.py
[2022-11-04 11:57:07,174] {logging_mixin.py:115} INFO - [2022-11-04 11:57:07,171] {dagbag.py:320} ERROR - Failed to import: /opt/airflow/dags/groups/group_downloads.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/groups/group_downloads.py", line 6
    with DAG(f"{parent_dag_id}.{child_dag_id")
                                             ^
SyntaxError: invalid syntax
[2022-11-04 11:57:07,189] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/groups/group_downloads.py
[2022-11-04 11:57:07,550] {processor.py:161} INFO - Processing /opt/airflow/dags/groups/group_downloads.py took 0.440 seconds
[2022-11-04 11:57:37,813] {processor.py:153} INFO - Started process (PID=1780) to work on /opt/airflow/dags/groups/group_downloads.py
[2022-11-04 11:57:37,816] {processor.py:641} INFO - Processing file /opt/airflow/dags/groups/group_downloads.py for tasks to queue
[2022-11-04 11:57:37,817] {logging_mixin.py:115} INFO - [2022-11-04 11:57:37,817] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/groups/group_downloads.py
[2022-11-04 11:57:37,826] {logging_mixin.py:115} INFO - [2022-11-04 11:57:37,825] {dagbag.py:320} ERROR - Failed to import: /opt/airflow/dags/groups/group_downloads.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/groups/group_downloads.py", line 6
    with DAG(f"{parent_dag_id}.{child_dag_id")
                                             ^
SyntaxError: invalid syntax
[2022-11-04 11:57:37,828] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/groups/group_downloads.py
[2022-11-04 11:57:37,853] {processor.py:161} INFO - Processing /opt/airflow/dags/groups/group_downloads.py took 0.044 seconds
[2022-11-04 11:58:36,441] {processor.py:153} INFO - Started process (PID=1813) to work on /opt/airflow/dags/groups/group_downloads.py
[2022-11-04 11:58:36,443] {processor.py:641} INFO - Processing file /opt/airflow/dags/groups/group_downloads.py for tasks to queue
[2022-11-04 11:58:36,445] {logging_mixin.py:115} INFO - [2022-11-04 11:58:36,445] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/groups/group_downloads.py
[2022-11-04 11:58:36,457] {logging_mixin.py:115} INFO - [2022-11-04 11:58:36,455] {dagbag.py:320} ERROR - Failed to import: /opt/airflow/dags/groups/group_downloads.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/groups/group_downloads.py", line 6
    with DAG(f"{parent_dag_id}.{child_dag_id")
                                             ^
SyntaxError: invalid syntax
[2022-11-04 11:58:36,458] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/groups/group_downloads.py
[2022-11-04 11:58:36,480] {processor.py:161} INFO - Processing /opt/airflow/dags/groups/group_downloads.py took 0.042 seconds
[2022-11-04 11:59:04,855] {processor.py:153} INFO - Started process (PID=1837) to work on /opt/airflow/dags/groups/group_downloads.py
[2022-11-04 11:59:04,857] {processor.py:641} INFO - Processing file /opt/airflow/dags/groups/group_downloads.py for tasks to queue
[2022-11-04 11:59:04,859] {logging_mixin.py:115} INFO - [2022-11-04 11:59:04,859] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/groups/group_downloads.py
[2022-11-04 11:59:04,871] {logging_mixin.py:115} INFO - [2022-11-04 11:59:04,869] {dagbag.py:320} ERROR - Failed to import: /opt/airflow/dags/groups/group_downloads.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/groups/group_downloads.py", line 9
    
    ^
SyntaxError: unexpected EOF while parsing
[2022-11-04 11:59:04,873] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/groups/group_downloads.py
[2022-11-04 11:59:04,899] {processor.py:161} INFO - Processing /opt/airflow/dags/groups/group_downloads.py took 0.049 seconds
[2022-11-04 11:59:21,213] {processor.py:153} INFO - Started process (PID=1861) to work on /opt/airflow/dags/groups/group_downloads.py
[2022-11-04 11:59:21,215] {processor.py:641} INFO - Processing file /opt/airflow/dags/groups/group_downloads.py for tasks to queue
[2022-11-04 11:59:21,218] {logging_mixin.py:115} INFO - [2022-11-04 11:59:21,218] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/groups/group_downloads.py
[2022-11-04 11:59:21,247] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/groups/group_downloads.py
[2022-11-04 11:59:21,266] {processor.py:161} INFO - Processing /opt/airflow/dags/groups/group_downloads.py took 0.059 seconds
[2022-11-04 11:59:37,347] {processor.py:153} INFO - Started process (PID=1871) to work on /opt/airflow/dags/groups/group_downloads.py
[2022-11-04 11:59:37,349] {processor.py:641} INFO - Processing file /opt/airflow/dags/groups/group_downloads.py for tasks to queue
[2022-11-04 11:59:37,351] {logging_mixin.py:115} INFO - [2022-11-04 11:59:37,351] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/groups/group_downloads.py
[2022-11-04 11:59:37,392] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/groups/group_downloads.py
[2022-11-04 11:59:37,424] {processor.py:161} INFO - Processing /opt/airflow/dags/groups/group_downloads.py took 0.083 seconds
[2022-11-04 12:00:07,992] {processor.py:153} INFO - Started process (PID=1904) to work on /opt/airflow/dags/groups/group_downloads.py
[2022-11-04 12:00:07,995] {processor.py:641} INFO - Processing file /opt/airflow/dags/groups/group_downloads.py for tasks to queue
[2022-11-04 12:00:07,997] {logging_mixin.py:115} INFO - [2022-11-04 12:00:07,997] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/groups/group_downloads.py
[2022-11-04 12:00:08,027] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/groups/group_downloads.py
[2022-11-04 12:00:08,061] {processor.py:161} INFO - Processing /opt/airflow/dags/groups/group_downloads.py took 0.076 seconds
[2022-11-04 12:00:12,355] {processor.py:153} INFO - Started process (PID=1910) to work on /opt/airflow/dags/groups/group_downloads.py
[2022-11-04 12:00:12,357] {processor.py:641} INFO - Processing file /opt/airflow/dags/groups/group_downloads.py for tasks to queue
[2022-11-04 12:00:12,359] {logging_mixin.py:115} INFO - [2022-11-04 12:00:12,358] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/groups/group_downloads.py
[2022-11-04 12:00:12,383] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/groups/group_downloads.py
[2022-11-04 12:00:12,413] {processor.py:161} INFO - Processing /opt/airflow/dags/groups/group_downloads.py took 0.064 seconds
[2022-11-04 12:00:18,408] {processor.py:153} INFO - Started process (PID=1920) to work on /opt/airflow/dags/groups/group_downloads.py
[2022-11-04 12:00:18,410] {processor.py:641} INFO - Processing file /opt/airflow/dags/groups/group_downloads.py for tasks to queue
[2022-11-04 12:00:18,412] {logging_mixin.py:115} INFO - [2022-11-04 12:00:18,412] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/groups/group_downloads.py
[2022-11-04 12:00:18,435] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/groups/group_downloads.py
[2022-11-04 12:00:18,452] {processor.py:161} INFO - Processing /opt/airflow/dags/groups/group_downloads.py took 0.049 seconds
[2022-11-04 12:00:21,512] {processor.py:153} INFO - Started process (PID=1921) to work on /opt/airflow/dags/groups/group_downloads.py
[2022-11-04 12:00:21,514] {processor.py:641} INFO - Processing file /opt/airflow/dags/groups/group_downloads.py for tasks to queue
[2022-11-04 12:00:21,515] {logging_mixin.py:115} INFO - [2022-11-04 12:00:21,515] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/groups/group_downloads.py
[2022-11-04 12:00:21,537] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/groups/group_downloads.py
[2022-11-04 12:00:21,554] {processor.py:161} INFO - Processing /opt/airflow/dags/groups/group_downloads.py took 0.045 seconds
[2022-11-04 12:00:23,534] {processor.py:153} INFO - Started process (PID=1922) to work on /opt/airflow/dags/groups/group_downloads.py
[2022-11-04 12:00:23,536] {processor.py:641} INFO - Processing file /opt/airflow/dags/groups/group_downloads.py for tasks to queue
[2022-11-04 12:00:23,537] {logging_mixin.py:115} INFO - [2022-11-04 12:00:23,537] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/groups/group_downloads.py
[2022-11-04 12:00:23,561] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/groups/group_downloads.py
[2022-11-04 12:00:23,580] {processor.py:161} INFO - Processing /opt/airflow/dags/groups/group_downloads.py took 0.050 seconds
[2022-11-04 12:00:44,625] {processor.py:153} INFO - Started process (PID=1945) to work on /opt/airflow/dags/groups/group_downloads.py
[2022-11-04 12:00:44,630] {processor.py:641} INFO - Processing file /opt/airflow/dags/groups/group_downloads.py for tasks to queue
[2022-11-04 12:00:44,634] {logging_mixin.py:115} INFO - [2022-11-04 12:00:44,634] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/groups/group_downloads.py
[2022-11-04 12:00:44,742] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/groups/group_downloads.py
[2022-11-04 12:00:44,809] {processor.py:161} INFO - Processing /opt/airflow/dags/groups/group_downloads.py took 0.213 seconds
[2022-11-04 12:00:58,933] {processor.py:153} INFO - Started process (PID=1954) to work on /opt/airflow/dags/groups/group_downloads.py
[2022-11-04 12:00:58,936] {processor.py:641} INFO - Processing file /opt/airflow/dags/groups/group_downloads.py for tasks to queue
[2022-11-04 12:00:58,938] {logging_mixin.py:115} INFO - [2022-11-04 12:00:58,938] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/groups/group_downloads.py
[2022-11-04 12:00:58,986] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/groups/group_downloads.py
[2022-11-04 12:00:59,024] {processor.py:161} INFO - Processing /opt/airflow/dags/groups/group_downloads.py took 0.105 seconds
[2022-11-04 12:01:29,532] {processor.py:153} INFO - Started process (PID=1987) to work on /opt/airflow/dags/groups/group_downloads.py
[2022-11-04 12:01:29,535] {processor.py:641} INFO - Processing file /opt/airflow/dags/groups/group_downloads.py for tasks to queue
[2022-11-04 12:01:29,539] {logging_mixin.py:115} INFO - [2022-11-04 12:01:29,539] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/groups/group_downloads.py
[2022-11-04 12:01:29,554] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/groups/group_downloads.py
[2022-11-04 12:01:29,589] {processor.py:161} INFO - Processing /opt/airflow/dags/groups/group_downloads.py took 0.070 seconds
[2022-11-04 12:01:59,949] {processor.py:153} INFO - Started process (PID=2020) to work on /opt/airflow/dags/groups/group_downloads.py
[2022-11-04 12:01:59,951] {processor.py:641} INFO - Processing file /opt/airflow/dags/groups/group_downloads.py for tasks to queue
[2022-11-04 12:01:59,953] {logging_mixin.py:115} INFO - [2022-11-04 12:01:59,953] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/groups/group_downloads.py
[2022-11-04 12:01:59,974] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/groups/group_downloads.py
[2022-11-04 12:02:00,011] {processor.py:161} INFO - Processing /opt/airflow/dags/groups/group_downloads.py took 0.067 seconds
[2022-11-04 12:02:30,442] {processor.py:153} INFO - Started process (PID=2046) to work on /opt/airflow/dags/groups/group_downloads.py
[2022-11-04 12:02:30,444] {processor.py:641} INFO - Processing file /opt/airflow/dags/groups/group_downloads.py for tasks to queue
[2022-11-04 12:02:30,446] {logging_mixin.py:115} INFO - [2022-11-04 12:02:30,446] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/groups/group_downloads.py
[2022-11-04 12:02:30,459] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/groups/group_downloads.py
[2022-11-04 12:02:30,486] {processor.py:161} INFO - Processing /opt/airflow/dags/groups/group_downloads.py took 0.048 seconds
